readme.md

```diff
@@ -116,7 +130,7 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+In a rebase you merge master into your feature branch to get out bugs there and then merge the feature branch back into master, this leads to a cleaner history on the master branch
 ```
```

Comment:
A rebase takes a set of commits, and re-writes them (modifies them) so that they appear to apply to a more recent point in history. This effectively makes it look like the rebased work was originally done on the commit it was rebased on, not the original commit it really was made on. In other words, the history appears to be liner, one change happening at a time, as opposed to being branched off.

A merge makes a new commit that describes how to combine the changes made in more than one branch. This preserves the history of how the work was done, but does lead to branches in your history.